### DISCUSSION TITLE
question not an issue: Vim compatibility?

### DISCUSSION THREAD

**Discussion by atsiporu** [CopilotC-Nvim/CopilotChat.nvim#139] [Q&A] [answered]
Is your plug-in a neovim specific or will it work with vim?

**Comment by jellydn** [comment]
> will it work with vim?

I haven't tested but I guess it might work with v2 (`canary` branch). Could you please try and let us know? Thanks.

**Reply by atsiporu** [reply]
So I tried. 
Unfortunately the plenary.nvim is Neovim specific and so becuase CopilotChat is dependent on plenary.nvim the CopilotChat is also Neovim dependent.
Thank you very much.

If you ever consider making your plugin VIM/Nvim independent ;) that would be totally awesome.
But anyway thanks for looking into it.

**Reply by deathbeam** [reply, marked as answer]
Yea even without plenary.nvim we use *a lot* of neovim specific APIs. There might be APIs that we could use from regular vim instead but it would be a lot of effort to support both, but atm we definitely do not support vim
